### ISSUE TITLE
Binary of app extension is used for coverage.

### ISSUE THREAD

**Issue by dhruvsbhatt** [SlatherOrg/slather#284]
I used to get coverage of my entire project, until I added app extension (Today Notification).
I've added the binary locations before and after adding extension.

```
Processing coverage file: /Users/dhruv.bhatt/Library/Developer/Xcode/DerivedData/Sop-dslnwitonjndytfmxmxynxavjjka/Build/Intermediates/CodeCoverage/Coverage.profdata
Against binary files:
	/Users/dhruv.bhatt/Library/Developer/Xcode/DerivedData/Sop-dslnwitonjndytfmxmxynxavjjka/Build/Intermediates/CodeCoverage/Products/Staging-iphonesimulator/Sop.app/Sop

```
Below is the path, after adding App Extension. Where the binary location changes and only few files are being covered.
```
Processing coverage file: /Users/dhruv.bhatt/Library/Developer/Xcode/DerivedData/Sop-aonjutxupcjztdeatmldgkcsraer/Build/Intermediates/CodeCoverage/Coverage.profdata
Against binary files:
	/Users/dhruv.bhatt/Library/Developer/Xcode/DerivedData/Sop-aonjutxupcjztdeatmldgkcsraer/Build/Intermediates/CodeCoverage/Products/Staging-iphonesimulator/Sop.app/PlugIns/Sop.appex/Sop
```
I tried adding file and base name parameters, but that didn't work.

**Comment by ksuther** [comment]
You could try specifying the path directly with `binary_file`. Also, have you tried using `scheme` so that slather can try to find the binary automatically?

**Comment by dhruvbhatt** [comment]
@ksuther Thank you.
What I did meanwhile to overcome the issue, I defined the path of derivedData. Then using 
`--build-directory` and  `--binary-file` , I somehow made it working.  And yes, I do've multiple schemes, so I do mention `scheme` as well.

The commands which I execute are : 
1) For Testing 
`xcodebuild test -workspace Sop.xcworkspace -scheme Sop-Staging -configuration Staging -destination 'platform=iOS Simulator,name=iPhone 7,OS=10.2' -enableCodeCoverage YES VALID_ARCHS=x86_64 ONLY_ACTIVE_ARCH=NO -derivedDataPath ../DerivedData | xcpretty -c "--report" "html" "--output" "../Project_report.html"`

2) Slathering
```
slather coverage --html --verbose --output-directory ../Coverage \
--workspace Sop.xcworkspace --scheme Sop-Staging \
--build-directory ../DerivedData \
--binary-file ../DerivedData/Build/Intermediates/CodeCoverage/Products/Staging-iphonesimulator/Sop.app/Sop Sop.xcodeproj
```
